xilem_web/src/events.rs

```diff
@@ -3,11 +3,13 @@
 
 use std::{borrow::Cow, marker::PhantomData};
 use wasm_bindgen::{prelude::Closure, throw_str, JsCast, UnwrapThrowExt};
-use web_sys::AddEventListenerOptions;
+use web_sys::{js_sys, AddEventListenerOptions};
 use xilem_core::{MessageResult, Mut, View, ViewId, ViewMarker, ViewPathTracker};
 
 use crate::{DynMessage, ElementAsRef, OptionalAction, ViewCtx};
 
+const ON_EVENT_VIEW_ID: ViewId = ViewId::new(0x2357_1113);
```

Comment:
Probably should have a doc comment here...

Reply:
I've added that in #493 (as I think a PR is overkill for this little change).